src/assets/avatars/o-avatar/o-avatar.component.ts

```diff
@@ -0,0 +1,15 @@
+import { CustomColorsIO } from '@/app/shared/customColors';
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-o-avatar',
+  templateUrl: './o-avatar.component.html',
+  styleUrls: ['./o-avatar.component.scss'],
+  standalone: true,
+})
+export class OAvatarComponent {
+  @Input() color: CustomColorsIO = CustomColorsIO.black; // Default color
+
+  height = 192;
+  width = 192;

```

Comment:
Remember to add this to the HTML aswell. Now it is defined both here and statically in the o-avatar html

Reply:
Yeah i know. Just pushed an unfinished solution, but will complete them later hehe